tests/test_integrity.py

```diff
@@ -476,6 +477,54 @@ def test_sources(db):
     assert len(t) == 37, f'found {len(t)} sources from {ref}'
 
 
+# importing table 9 for data in test parameters
+def test_ingest_modeledparameters(db):
+    # organized data
+    model_params_data = [{'source': '2MASS J00001354+2554180', 'value': 0.99, 'parameter': 'radius', 'reference': 'Fili15', 'unit': 'R_jup'},
+                         {'source': '2MASS J00001354+2554180', 'value': 5.02, 'parameter': 'log g', 'reference': 'Fili15', 'unit': 'dex'},
+                         {'source': '2MASS J00001354+2554180', 'value': 1227.0, 'parameter': 'T eff', 'reference': 'Fili15', 'unit': 'K'},
+                         {'source': '2MASS J00001354+2554180', 'value': 47.56, 'parameter': 'mass', 'reference': 'Fili15', 'unit': 'M_jup'},
+                         # next data points should fail
+                         {'source': '2MASS J00034227-2822410', 'value': 106.97, 'parameter': 'mass', 'reference': 'Fili15', 'unit': 'M_Jupiter'},
+                         {'source': '2MASS J00034227-2822410', 'value': 1.32, 'parameter': 'radius', 'reference': 'Fili15', 'unit': 'R_Jup'},
+                         {'source': '2MASS J00034227-2822410', 'value': 2871.0, 'parameter': 'T eff', 'reference': 'Fili15', 'unit': 'kelvin'},
+                         {'source': '2MASS J00034227-2822410', 'value': 5.18, 'parameter': 'log g', 'reference': 'Fili15', 'unit': ''}]
+
+    db.ModeledParameters.insert().execute(model_params_data)
+
+
+
+
+def test_modeledparameters(db):
+
+
+    # There should be no entries in the modeled parameters table without parameter
+
+    t = db.query(db.ModeledParameters.c.source). \
+         filter(db.ModeledParameters.c.parameter.is_(None)). \
+         astropy()
+    #
+    if len(t) > 0:
+         print('\nEntries found without a parameter')
+         print(t)
+    assert len(t) == 0
+
+    # Test units are astropy.unit resolvable
+    t = db.query(db.ModeledParameters.c.unit).filter(db.ModeledParameters.c.unit.is_not(None)).distinct().astropy()
+    unit_fail = []
+    for x in t:
+        unit = x['unit']
+
+        try:
+            assert u.Unit(unit, parse_strict='raise')
+        except ValueError:
+            print(f'{unit} is not a recognized astropy unit')
+            counts = db.query(db.ModeledParameters).filter(db.ModeledParameters.c.unit == unit).count()
+            unit_fail.append({unit: counts})  # count of how many of that unit there is
+
+    assert len(unit_fail) == 0, f'Some parameter units did not resolve: {unit_fail}'
```

Comment:
I think this is the line that may need to change give the units we expect to fail.  Maybe `len(unit_fail) == 4`.